packages/core/src/awsService/appBuilder/serverlessLand/main.ts

```diff
@@ -95,20 +93,28 @@ async function launchProjectCreationWizard(
     }).run()
 }
 
-async function downloadPatternCode(config: CreateServerlessLandWizardForm, assetName: string): Promise<void> {
+export async function downloadPatternCode(config: CreateServerlessLandWizardForm, assetName: string): Promise<void> {
     const fullAssetName = assetName + '.zip'
     const location = vscode.Uri.joinPath(config.location, config.name)
     try {
         await getPattern(serverlessLandOwner, serverlessLandRepo, fullAssetName, location, true)
+        telemetry.record({
```

Comment:
What is the metric we are recording to? This makes assumptions that the caller of this function is using a `telemetry.XXXXX.run()`, but we cannot be sure. Unless you have a good reason, we recommend passing the `span` object created by `run()` as an argument and then using `record()` on that

Reply:
We're trying to record the `pattern` that user has selected to download. This way we can keep track of the most downloaded pattern, along with the IaC and Runtime.

Reply:
This is one of the crucial metrics that we want to record for this project

Reply:
Ok I see that it is recording to the `aws.toolkit.lambda.createServerlessLandProject` command metric